config/routes.rb

```diff
@@ -14,4 +14,9 @@
 
   get "/course/:provider_code/:course_code", to: "courses#show", as: "course"
   get "/results", to: "results#index", as: "results"
+
+  scope "/results/filter" do
```

Comment:
I think...

```suggestion
  scope module: "filters", path: "/results/filter" do
```
will save you having to `to: "filters/`